### DISCUSSION TITLE
grid-node download the default chrome driver at startup

### DISCUSSION THREAD

**Discussion by pansfy** [seleniumbase/SeleniumBase#844] [Q&A] [answered]
The driver that matches the version has been downloaded,but sbase grid-node start --hub=127.0.0.1 will download the default chrome driver of version 2.44.
How to solve this problem?
Thanks.

**Comment by mdmintz** [comment, marked as answer]
Hi @pansfy , when using the Grid, a copy of chromedriver must also be placed on the path. This can be done by adding ``--path`` to the driver install command on Mac/Linux. For Windows, you'll have to copy/paste the driver manually. And if you already have an old driver on your system path at a different location, you may need to remove that manually first. Here's the command that adds the latest chromedriver to the system path on Mac/Linux:

```bash
seleniumbase install chromedriver latest --path

*** chromedriver version for download = 89.0.4389.23 (Latest)

Downloading chromedriver_mac64.zip from:
https://chromedriver.storage.googleapis.com/89.0.4389.23/chromedriver_mac64.zip ...
Download Complete!

Extracting ['chromedriver'] from chromedriver_mac64.zip ...
Unzip Complete!

The file [chromedriver] was saved to:
/Users/michael/github/SeleniumBase/seleniumbase/drivers/chromedriver

Making [chromedriver 89.0.4389.23] executable ...
[chromedriver] is now ready for use!
Also copied to: /usr/local/bin/chromedriver
```

**Reply by pansfy** [reply]
yes,thanks.i solved the problem.
